package/skelcd-control-SLED.spec

```diff
@@ -36,7 +36,8 @@ Group:          Metapackages
 BuildRequires:  libxml2-tools
 # Added software/default_modules
 BuildRequires:  yast2-installation-control >= 4.2.9
-BuildRequires:  sle15-desktop-migration
+# To migrate old KDE desktop
+Requires:  sle15-desktop-migration
```

Comment:
We need to add changes and bump the version, otherwise this change won't be autosubmitted to SP3...